pkg/rtc/supervisor/publication_monitor.go

```diff
@@ -82,6 +84,8 @@ func (p *PublicationMonitor) addPending() {
 	)
 	p.update()
 	p.lock.Unlock()
+
+	prometheus.AddPublishAttempt(trackType)
```

Comment:
Same comment as the subscribe case. If you want to ensure that the `attempt` counter increments before `success` counter, you can move this before call to `update()` above